sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -470,6 +465,18 @@ object Radar {
 
         this.initialized = true
 
+        sendLogConversionRequest(
+            "app_open",
+            callback = object : RadarLogConversionCallback {
+                override fun onComplete(
+                    status: RadarStatus,
+                    event: RadarEvent?
+                ) {
+                    this@Radar.logger.i("Conversion name = ${event?.conversionName}: status = $status; event = $event")
+                }
+            }
+        )
```

Comment:
We should move this here: https://github.com/radarlabs/radar-sdk-android/blob/master/sdk/src/main/java/io/radar/sdk/RadarActivityLifecycleCallbacks.kt#L40

Reply:
yup, will do